src/Language/PureScript/Bundle.hs

```diff
@@ -279,16 +254,89 @@ toModule requirePath mids mid top
     exportType (JSIdentifier s) = pure (RegularExport s)
     exportType _ = err UnsupportedExport
 
-    extractLabel :: Node -> m String
-    extractLabel (JSStringLiteral _ nm) = pure nm
-    extractLabel (JSIdentifier nm) = pure nm
-    extractLabel _ = err UnsupportedExport
-
-    isSeparator :: Node -> Bool
-    isSeparator (JSLiteral ",") = True
-    isSeparator _ = False
   toModuleElement other = pure (Other other)
 
+-- | Get all the exported identifiers from an FFI module.
+getExportedIdentifiers :: forall m.
+  (Applicative m, MonadError ErrorMessage m) =>
+  String
+  -> JSNode
+  -> m [String]
+getExportedIdentifiers mname top
+  | JSSourceElementsTop ns <- node top = concat <$> traverse go ns
+  | otherwise = err InvalidTopLevel
+  where
+  err = throwError . ErrorInModule (ModuleIdentifier mname Foreign)
+
+  go :: JSNode -> m [String]
+  go n
+    | Just props <- matchExport n
+    = traverse toIdent props
+    | Just (True, name, _) <- matchMember n
```

Comment:
Ok, looking at this again, I think I've worked out the answers: a Member is just a declaration inside an FFI module: either `exports.foo = bar` or `var foo = bar` at the top level, and the `Bool` argument does indeed seem to indicate whether or not it is exported.